tests/users/test_signup_view.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth.models import User
```

Comment:
The file name should be test_signup_views

Reply:
Changed